pilot/pkg/config/kube/ingress/conversion.go

```diff
@@ -90,6 +90,10 @@ func ConvertIngressV1alpha3(ingress v1beta1.Ingress, mesh *meshconfig.MeshConfig
 	}
 
 	for i, tls := range ingress.Spec.TLS {
+		if tls.SecretName == "" {
```

Comment:
- If secret is not defined in the K8s Ingress, but TLS section is present, it should use default secret that is defined in the  controller deployment, and if there a secret, use it instead of default.
- if there is no default secret and no secret in the Ingress resource - log warning.

Reply:
I think the default secret is something nginx made up, we try not to do anything special with our Ingress implementation because we have Gateway/VirtualService for advanced cases